iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/HelmProcessor.java

```diff
@@ -19,24 +19,78 @@
  */
 package org.sonar.iac.kubernetes.plugin;
 
-import java.util.regex.Pattern;
+import java.io.IOException;
+import javax.annotation.Nullable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.sonar.api.ExtensionPoint;
 import org.sonar.api.scanner.ScannerSide;
+import org.sonar.iac.common.extension.visitors.InputFileContext;
+import org.sonar.iac.helm.HelmEvaluator;
+import org.sonar.iac.helm.jna.Loader;
+import org.sonar.iac.helm.jna.library.IacHelmLibrary;
 import org.sonarsource.api.sonarlint.SonarLintSide;
 
+import static org.sonar.iac.helm.utils.HelmFilesystemUtils.findValuesFile;
+
 @ScannerSide
 @SonarLintSide
 @ExtensionPoint
 public class HelmProcessor {
-
   private static final Logger LOG = LoggerFactory.getLogger(HelmProcessor.class);
-  private static final Pattern HELM_CODE = Pattern.compile("\\{\\{[^{]*\\}\\}");
+  @Nullable
+  private HelmEvaluator helmEvaluator;
+  protected Loader loader = new Loader();
+
+  public HelmProcessor() {
+  }
+
+  // for testing
+  HelmProcessor(HelmEvaluator helmEvaluator) {
+    this.helmEvaluator = helmEvaluator;
+  }
+
+  public void initialize() {
+    HelmEvaluator newHelmEvaluator;
+    try {
+      IacHelmLibrary library = loader.load("/sonar-helm-for-iac", IacHelmLibrary.class);
+      newHelmEvaluator = new HelmEvaluator(library);
+    } catch (RuntimeException e) {
+      LOG.debug("Native library not loaded, Helm integration will be disabled", e);
```

Comment:
I would not put that information in `debug` but more likely in `info` or `warn`. Looks like an important piece of information, and if a user is complaining that helm is not working, I would not want to risk missing it because `debug` log is disabled.